doc/source/tutorial/signal.rst

```diff
@@ -1048,8 +1048,598 @@ giving a factor of :math:`\sim 2` speed increase over the straightforward
+    >>> Sz1 = SFT.stft(z)
+    ...
+    >>> # Plot results:
+    >>> fig1, axx = plt.subplots(2, 1, sharex='all', sharey='all',
+    ...                          figsize=(6., 5.))  # enlarge figure a bit
+    >>> t_lo, t_hi, f_lo, f_hi = SFT.extent(N, center_bins=True)
+    >>> t_str0 = r"Legacy stft() produces $%d\times%d$ points" % Sz0.T.shape
+    >>> t_str1 = r"ShortTimeFFT produces $%d\times%d$ points" % Sz1.T.shape
+    >>> _ = axx[0].set(title=t_str0, xlim=(t_lo, t_hi), ylim=(f_lo, f_hi))
+    >>> _ = axx[1].set(title=t_str1, xlabel="Time $t$ in seconds " +
+    ...                rf"($\Delta t= %g\,$s)" % SFT.delta_t)
+    ...
+    >>> # Calculate extent of plot with centered bins since imshow
+    ... # does not interpolate by default:
+    ... dt2 = (nperseg-noverlap) / fs  / 2 # equals SFT.delta_t / 2
+    >>> df2 = fs / nperseg / 2 # equals SFT.delta_f / 2
+    >>> extent0 = (-dt2, t0[-1] + dt2, f0[0] - df2, f0[-1] - df2)
+    >>> extent1 = SFT.extent(N, center_bins=True)
+    ...
+    >>> kw = dict(origin='lower', aspect='auto', cmap='viridis')
+    >>> im1a = axx[0].imshow(abs(Sz0), extent=extent0, **kw)
+    >>> im1b = axx[1].imshow(abs(Sz1), extent=extent1, **kw)
+    >>> fig1.colorbar(im1b, ax=axx, label="Magnitude $|S_z(t, f)|$")
+    >>> _ = fig1.supylabel(rf"Frequency $f$ in Hertz ($\Delta f = %g\,$Hz)" %
+    ...                    SFT.delta_f)
+    >>> plt.show()
+
+
+That the |ShortTimeFFT| produces 3 more time slices than the legacy version is
+the main difference. As laid out in the :ref:`tutorial_stft_sliding_win`
+section, all slices which touch the signal are incorporated in the new version.
+This has the advantage that the STFT can be sliced and reassembled as shown in
+the |ShortTimeFFT| code example. Furthermore, using all touching slices makes
+the ISTFT more robust in the case the windows are not non-zero everywhere.
+
+Note that the slices with identical time stamps produce equal results
+(up to numerical accuracy), i. e.:
+
+    >>> np.allclose(Sz0, Sz1[:, 2:-1])
+    True
+
+Generally, those the additional slices contain non-zero values. Due to the
+large overlap in our example, they are quite small. E.g.:
+
+    >>> abs(Sz1[:, 1]).min(), abs(Sz1[:, 1]).max()
+    (6.925060911593139e-07, 8.00271269218721e-07)
+
+The ISTFT can be utilized to reconstruct the original signal:
+
+    >>> t0_r, z0_r = istft(Sz0_u, fs, win, nperseg, noverlap,
+    ...                    input_onesided=False, scaling='spectrum')
+    >>> z1_r = SFT.istft(Sz1, k1=N)
+    ...
+    >>> len(z0_r), len(z)
+    (1010, 1001)
+    >>> np.allclose(z0_r[:N], z)
+    True
+    >>> np.allclose(z1_r, z)
+    True
+
+Note that the legacy implementation returns a signal, which is longer than the
+original. On the other hand, the new `istft` allows the length signal length or
+the slice to be specified. The deviation in length in the old implementation is
+caused by the fact the signal length is not a multiple of the slices.
```

Comment:
```suggestion
the slice to be specified. The length discrepancy in the old implementation is
caused by the fact that the signal length is not a multiple of the slices.
```

"the new `istft` allows the signal length or the slice to be specified" meaning the k0, k1 parameter?